spec/Section 6 -- Execution.md

```diff
@@ -492,9 +496,12 @@ BatchIncrementalResults(incrementalResults):
 
 ## Executing an Execution Plan
 
-To execute a execution plan, the object value being evaluated and the object
-type need to be known, as well as whether the non-deferred grouped field set
-must be executed serially, or may be executed in parallel.
+Executing an execution plan consists of two tasks that may be performed in
```

Comment:
Move to after execution plan generation?

Reply:
still not sure about the ordering of these sections. technically, as we've spoken about offline, once we took up @benjie 's changes in #1039, CollectFIelds() is in the wrong order, and then I am now sure about how to best do the rest of the ordering.

Reply:
I think we can leave as is for now and readdress once #1039 is merged.